integration/experiment/power_sweep/power_sweep.py

```diff
@@ -41,6 +41,21 @@
 import geopmpy.io
 
 from experiment import launch_util
+from experiment import common_args
+
+def setup_run_args(parser, cool_off_time=60):
+    common_args.add_output_dir(parser)
```

Comment:
is this deliberately not using common_args.setup_run_args()?

Reply:
if different defaults is the problem, it could forward them to that function (maybe a dict?)

Reply:
promoting the cool_off_time default to be locally configured is the issue that cause the change in the pattern.  I'm not sure its is a huge problem as it is.